code/modules/client/client procs.dm

```diff
@@ -381,6 +381,21 @@
 	spawn()
 		if(src in clients) //Did we log out before we reached this part of the function?
 			nanomanager.send_resources(src)
+	// Hahaha your suffering is not yet over, as we still must waterboard you with ten thousand oggs! (4500 at last count)
+	spawn()
+		if(src)
```

Comment:
iirc there's no way for this to fail other than manually setting `src = null`
when the datum gets deleted its scheduled procs just get removed from the queue

Reply:
Actually, during testing, it would fail if you logged during loading.

Reply:
I guess clients work differently